README.md

```diff
@@ -26,7 +26,7 @@ sudo npm run patch:hosts
 
 1. ```npm install```
 
-2. ```PROXY=true npm run start:beta```
+2. ```npm run start``` choose stage and then beta
```

Comment:
I believe you need the PROXY=true for the ui to pull chrome and all the other consoleDot components, don't you?